scripts/test_geocoder_production.py

```
@@ -0,0 +1,190 @@
+        if 'nominatim_circuit_breaker_state' in stats:
+            print(f"Nominatim circuit: {stats['nominatim_circuit_breaker_state']}")
+        
+        if 'google_circuit_breaker_state' in stats:
+            print(f"Google circuit: {stats['google_circuit_breaker_state']}")
+        
+        if 'google_daily_quota_used' in stats:
+            print(f"Google quota used: {stats['google_daily_quota_used']}")
+        
+        print(f"\\n✅ Production test completed!")
+        
+    except ImportError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)